### PR TITLE
Implement enable/disable Animations

### PR DESCRIPTION
**Pull request by vitormateusalmeida**
Add an option to enable/disable animations. Toast animation is broken (should be fixed in another issue, imo). Issue #5846

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ public:
     askForFeedbackOnLogout: false
     defaultSettings:
       application:
+        animations: false
```
Comment (Tainan404) [inline bigbluebutton-html5/private/config/settings.yml]: I think animations should be enable by default.

File: bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss [hunk 2]
```diff
@@ -25,4 +25,6 @@
   --indicator-padding-bottom: 0.7em;
 
   --user-indicators-offset: -5px;
+
+  --enableAnimation: 1;
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss]: Why are you overwriting this variable on modal style? https://github.com/bigbluebutton/bigbluebutton/pull/6634/files#diff-3b7d67cedd4b25db05c46bbaefd0a23dR2

**Comment by Tainan404** [comment]
When speaking without animation the presentation jumps.

peek 2019-01-28 11-19

File: bigbluebutton-html5/imports/ui/components/tooltip/component.jsx [hunk 3]
```diff
@@ -56,10 +61,29 @@ class Tooltip extends Component {
       distance: 10,
       arrow: true,
       arrowType: 'sharp',
+      animation: (enableAnimation) ? 'shift-away' : 'none',
     };
     this.tooltip = Tippy(`#${this.tippySelectorId}`, options);
   }
 
+  componentDidUpdate() {
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/tooltip/component.jsx]:
```suggestion
 componentDidUpdate() {
  const { enableAnimation } = this.state;
  const { animations } = Settings.application;

  if (animations !== enableAnimation) {
    const elements = document.querySelectorAll('[id^="tippy-"]');
    elements.forEach((e) => {
      const instance = e._tippy;
      instance.set({ animation: animations ? 'shift-away' : 'none' });
    });
    this.setEnableAnimation(animations);
  }
}
```

File: bigbluebutton-html5/imports/ui/components/join-handler/component.jsx [hunk 4]
```diff
@@ -33,6 +34,10 @@ class JoinHandler extends Component {
 
   componentDidMount() {
     this.fetchToken();
+
+    const { animations } = Settings.application;
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/join-handler/component.jsx]: That should be a callback from toggle.

**Comment by KDSBrowne** [comment]
Didn't notice the comment in the description to fix the toasts in another PR.

**Pull request merged**